src/DependencyInjection/Configuration.php

```diff
@@ -121,6 +121,7 @@ private function addContextsSection(ArrayNodeDefinition $node)
                                         ->scalarNode('format')->defaultValue('jpg')->end()
                                         ->scalarNode('constraint')->defaultValue(true)->end()
                                         ->scalarNode('resizer')->defaultNull()->end()
+                                        ->arrayNode('resizer_options')->defaultValue([])->useAttributeAsKey('name')->prototype('scalar')->end()
```

Comment:
Please add ->info() and explain what it does

Reply:
done.